error_pages/404.php

```diff
@@ -2,118 +2,131 @@
+				<?php wdnInclude("/wdn/templates_5.0/includes/global/apply-global-1.html"); ?>
+				<!-- TemplateBeginEditable name="applylocal" -->
+				<?php wdnInclude("/wdn/templates_5.0/includes/local/apply-local.html"); ?>
 				<!-- TemplateEndEditable -->
-			</nav>
-			<div id="wdn_navigation_wrapper">
-				<nav id="navigation" role="navigation" aria-label="main navigation">
-					<!-- TemplateBeginEditable name="navlinks" -->
-					<?php include "$root/sharedcode/navigation.php"; ?>
+				<?php wdnInclude("/wdn/templates_5.0/includes/global/apply-global-2.html"); ?>
+				<?php wdnInclude("/wdn/templates_5.0/includes/global/give-global-1.html"); ?>
+				<!-- TemplateBeginEditable name="givelocal" -->
+				<?php wdnInclude("/wdn/templates_5.0/includes/local/give-local.html"); ?>
+				<!-- TemplateEndEditable -->
+				<?php wdnInclude("/wdn/templates_5.0/includes/global/give-global-2.html"); ?>
+		<?php wdnInclude("/wdn/templates_5.0/includes/global/nav-global-2.html"); ?>
+		<?php wdnInclude("/wdn/templates_5.0/includes/global/idm.html"); ?>
+		<?php wdnInclude("/wdn/templates_5.0/includes/global/search.html"); ?>
+		<?php wdnInclude("/wdn/templates_5.0/includes/global/header-global-2.html"); ?>
+		<?php wdnInclude("/wdn/templates_5.0/includes/global/logo-lockup-1.html"); ?>
+		<?php wdnInclude("/wdn/templates_5.0/includes/global/site-affiliation-1.html"); ?>
+	<!-- TemplateBeginEditable name="affiliation" -->
+	<!-- TemplateEndEditable -->
+		<?php wdnInclude("/wdn/templates_5.0/includes/global/site-affiliation-2.html"); ?>
+		<?php wdnInclude("/wdn/templates_5.0/includes/global/site-title-1.html"); ?>
+	<!-- TemplateBeginEditable name="titlegraphic" -->
+	<a class="unl-site-title-medium" href="/">VITA Lincoln</a>
+	<!-- TemplateEndEditable -->
+		<?php wdnInclude("/wdn/templates_5.0/includes/global/site-title-2.html"); ?>
+		<?php wdnInclude("/wdn/templates_5.0/includes/global/logo-lockup-2.html"); ?>
+		<?php wdnInclude("/wdn/templates_5.0/includes/global/nav-toggle-group.html"); ?>
+		<?php wdnInclude("/wdn/templates_5.0/includes/global/nav-menu-1.html"); ?>
+		<?php wdnInclude("/wdn/templates_5.0/includes/global/nav-toggle-btn.html"); ?>
+		<?php wdnInclude("/wdn/templates_5.0/includes/global/nav-menu-child-1.html"); ?>
+	<!-- TemplateBeginEditable name="navlinks" -->
+		<?php include "$root/sharedcode/navigation.php"; ?>
+	<!-- TemplateEndEditable -->
+		<?php wdnInclude("/wdn/templates_5.0/includes/global/nav-menu-child-2.html"); ?>
+		<?php wdnInclude("/wdn/templates_5.0/includes/global/nav-menu-2.html"); ?>
+</header>
+
+<main class="dcf-main" id="dcf-main" role="main" tabindex="-1">
+
+	<!-- TemplateBeginEditable name="hero" -->
+	<div class="dcf-hero dcf-hero-default">
+		<!-- TemplateEndEditable -->
+		<div class="dcf-hero-group-1">
+			<div class="dcf-breadcrumbs-wrapper">
+				<nav class="dcf-breadcrumbs" id="dcf-breadcrumbs" role="navigation" aria-label="breadcrumbs">
```

Comment:
Not that it really matters, but do we have to specify the `id` the same as the `class` on these? The same thing was done on the `header` and `page-title` elements as well.

Reply:
Again, I chose to leave these elements as they were in the template files.

https://github.com/unl/wdntemplates/blob/4767842e3efd6c537079a75481708ec7de7495a0/Templates/php.fixed.dwt.php#L82

https://github.com/unl/wdntemplates/blob/4767842e3efd6c537079a75481708ec7de7495a0/Templates/php.fixed.dwt.php#L92